rma/README.rst

```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+RMA
+===
+
+You use this module to manage RMA (Return Merchandise Authorization) in Odoo
+
+The RMA can be send from the company to the customer or from the supplier to
+the company
+
+The RMA user creates the RMA and assign a responsible by selecting the
+affected invoice. The available operations are refund, repair and deliver or
+replace.
+The assigned person checks if the conditions for the RMA are correct and
+approve the RMA
+
+The assigned person have to go to the RMA lines from the RMA and operate. THe
+available options are creating receipts, delivery orders and refunds.
+
+Once the assigned person considers that the RMA is settled he/she can
+set the RMA to Done.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to Settings > Users and add the user to the group RMA manager.
+#. Go to Inventory > Settings > Return Merchandising Authorization and select
+   the option "Display 3 fields on rma: partner, invoice address, delivery
+   address" if needed.
+#. Go to Inventory > Settings > Configuration > Warehouse management >
+   Warehouses and add a default RMA location and RMA picking type for customers
+   and suppliers RMA picking type. It's very important to select the type of
+   operation supplier if we are moving in the company and customer if we are
+   moving out of the company.
```

Comment:
This option is only configurable if you config your warehouse to use routes, if not, then default configuration is ok (is ok after fix demo and data xml)